application/forms/RotationConfigForm.php

```diff
@@ -1431,6 +1431,7 @@ private function yieldRecurrenceRules(int $count): Generator
                 if (
                     $fromDay < $toDay && ($firstHandoffDay < $fromDay || $firstHandoffDay > $toDay)
                     || $toDay < $fromDay && ($firstHandoffDay < $fromDay && $firstHandoffDay > $toDay)
+                    || $firstHandoffDay === $toDay && $options["to_at"] === "00:00"
```

Comment:
That's the wrong fix as discussed. The previous error, the entry without width, must be fixed so that the entry has a width, and does not disappear.